include/pmix_version.h.in

```diff
@@ -17,4 +18,6 @@
 #define PMIX_VERSION_MAJOR @pmixmajor@
 #define PMIX_VERSION_MINOR @pmixminor@
 #define PMIX_VERSION_RELEASE @pmixrelease@
+
+#define PMIX_NUMERIC_VERSION 0x00040000
```

Comment:
Is this meant to be a single number to separate the versions (for quick compare)?
Do you want it to be generated based upon the `VERSION` file?

Reply:
Please see #792 for explanation of the fields. I wasn't sure how to autogenerate it, but that would clearly be preferable if you can suggest the method.